lib/SILOptimizer/Transforms/SILMem2Reg.cpp

```diff
@@ -415,12 +447,16 @@ StackAllocationPromoter::promoteAllocationInBlock(SILBasicBlock *BB) {
         // If we are loading from the AllocStackInst and we already know the
         // content of the Alloca then use it.
         LLVM_DEBUG(llvm::dbgs() << "*** Promoting load: " << *Load);
-        
         replaceLoad(Load, RunningVal, ASI);
         ++NumInstRemoved;
```

Comment:
Question. Shouldn't there be a continue here since we have replaced something and should just process the next inst? Or do we need to use the code below?

Reply:
There is a `continue` in the outer `if`. I'll add it here for clarity.